# config/v2x_gw.param.yaml
---
# Node-specific parameters
/**:
  ros__parameters:
    server:
      udp_ip_tx_: "127.0.0.1" # UDP target IP
      rx_udp_port: 5556 # UDP port to receive ASN.1 data
      tx_udp_port: 5555 # UDP port to send ASN.1 data
      cycle_time_ms: 10 # cycle time leads to communication update interval

    header:
      station_id: 1337 # this value is pseudonymized and should be unique in a GN communication (used to generate DENM)

    cam:
      active: true # activate processing of CAM module
      handler_debug_msg_throttle_ms: 5000 # throttle for debug msgs in milliseconds (throttles <=WARN)

    cpm:
      active: true # activate processing of CPM module
      handler_debug_msg_throttle_ms: 5000 # throttle for debug msgs in milliseconds (throttles <=WARN)

    denm:
      active: true # activate processing of DENM module
      handler_debug_msg_throttle_ms: 5000 # throttle for debug msgs in milliseconds (throttles <=WARN)

    examplem:
      active: false # activate processing of ExampleM module
      handler_debug_msg_throttle_ms: 5000 # throttle for debug msgs in milliseconds (throttles <=WARN)
